docs/classic-ui/views.md

```diff
@@ -820,9 +820,33 @@ You need to get access to the view in your code if you are:
 -   calling a view from inside another view, or
 -   calling a view from your unit test code.
 
-Below are two different approaches for that.
+Below are tree different approaches for that.
 
 
+(classic-ui-view-by-plone-api)
+### By using plone.api.content.get_view()
+
+The plone.api provides a method to get a view by it's registered name, the context and the current request.
```

Comment:
```suggestion
The `plone.api` provides a method to get a view by its registered name, the context, and the current request.
```

Reply:
FYI, "it's" is a contraction of "it is". "its" is the possessive form of the pronoun "it".